src/bp-activity/bp-activity-filters.php

```diff
@@ -178,6 +178,7 @@
 add_filter( 'bp_get_activity_content', 'bb_mention_add_user_dynamic_link', 20, 1 );
 add_filter( 'bp_get_activity_content_body', 'bb_mention_add_user_dynamic_link', 20, 1 );
 add_filter( 'bp_activity_comment_content', 'bb_mention_add_user_dynamic_link', 20, 1 );
+add_filter( 'the_content', 'bb_mention_add_user_dynamic_link', 20, 1 );
```

Comment:
@rajinsharwar2 
We have class file for woocommerce. Here os the path - `src/bp-core/compatibility/class-bb-woocommerce-helpers.php`. You should move this code inside above file.

Reply:
Updated!